dev-libs/imath/metadata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>waebbl-gentoo@posteo.net</email>
+		<name>Bernd Waibel</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<description>Gentoo Proxy Maintainers</description>
+	</maintainer>
+	<longdescription>
+	Basic, light-weight C++ library for efficient representation of 2D
+	and 3D vectors and matrices and data type common to computer graphics
+	applications.
+	</longdescription>
+	<use>
+		<flag name="large-stack">
+			Enables code to take advantage of large stack support
+		</flag>
+	</use>
+	<upstream>
+		<remote-id type="github">AcademySoftwareFoundation/Imath</remote-id>
```

Comment:
What about adding
<doc>https://imath.readthedocs.io/en/latest/</doc>
and using github address for HOMEPAGE?